<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_vertex">
       <property name="text">
        <string>PROJECTION TYPE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="button_drawing_type_projection">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <item>
        <property name="text">
         <string> parallel projection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>central projection</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_edge">
       <property name="text">
        <string>EDGE SETTING</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="button_drawing_type_edge">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>solid </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dotted</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_enge">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>on</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_color_edge">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ðŸ–Œ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="button_size_edge">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>VERTEX SETTING</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="button_drawing_type_vertex">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>circle </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>square </string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_vertex">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>on</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_color_vertex">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ðŸ–Œ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="button_size_vertex">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>BACKGROUND SETTING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_backround_color">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>color ðŸ–Œ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
